test/buffer_manager_tests.cc

```diff
@@ -1,5 +1,6 @@
 #include "sycl_wrappers.h"
 
+#include "catch2/matchers/catch_matchers_string.hpp"
```

Comment:
Should probably use angle brackets <>.

Reply:
The one time I decide to use the clang-tidy auto fix...